server/lib/state.py

```
@@ -0,0 +1,103 @@
+GAME_EXPIRATION_TIME = 86_400  # 1 day
+ROOM_EXPIRATION_TIME = 86_400  # 1 day
+
+
+class State:
+    def __init__(self):
+        self.redis = Redis.from_url(REDIS_URL)
+
+    def allow_player(self, action: str, room: str, player: Player) -> (bool, Optional[str]):
+        # Validate player
+        if not player.name or player.name == '':
+            return (False, f'name cannot be blank')
```

Ruff (0.13.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)